src/main/java/kr/hhplus/be/server/ranking/service/GoodsRankingService.java

```diff
@@ -0,0 +1,47 @@
+package kr.hhplus.be.server.ranking.service;
+
+import kr.hhplus.be.server.ranking.domain.GoodsRankDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.redis.core.StringRedisTemplate;
+import org.springframework.data.redis.core.ZSetOperations;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+@Service
+@RequiredArgsConstructor
+public class GoodsRankingService {
+
+    private static final String RANKING_KEY = "goods:ranking";
+
+    private final int rankCount = 10;
```

Comment:
```suggestion
@Value("${ranking.top-n:10}")
private int rankCount;
```
application.yml와 같은 설정파일에 상수값을 작성하고
@Value 어노테이션을 사용하면 관리가 편리하고 쉽게 수정할수 있습니다.